blockchain/genesis/genesis.go

```diff
@@ -178,6 +179,8 @@ type (
 		NumDelegates uint64 `yaml:"numDelegates"`
 		// NumCandidateDelegates is the number of candidate delegates, who may be selected as a delegate via roll dpos
 		NumCandidateDelegates uint64 `yaml:"numCandidateDelegates"`
+		// NumCandidateDelegatesV2 is the number of candidate delegates after extension
+		NumCandidateDelegatesV2 uint64 `yaml:"numCandidateDelegatesV2"`
```

Comment:
naming with hard-fork name for better readability

Reply:
the current way in the codebase is to use `V2`